src/pages/about.svelte

```diff
@@ -3,11 +3,17 @@
 <CountersSection/>
 <ReferencesSection/>
 <ToolsSection/>
+<Meta {metadata}/>
```

Comment:
Maybe there's a way to add this `<Meta/>` to `pages/_layout.svelte` instead of adding it to all pages? Maybe something like this? https://stackoverflow.com/questions/61960147/how-to-pass-data-from-child-to-parent-with-2-svelte-components-in-html-parent-ch